GNUmakefile

```diff
@@ -194,6 +194,18 @@ test-ci: other-consul dev-build vet test-install-deps
 	    fi \
 	fi
 
+test-flake: other-consul vet test-install-deps
+	@ if [ -z $(ARGS) ] ; then \
+		echo ""; \
+		echo "error: ARGS required for flake-repro"; \
+		echo "ex: make test-flake ARGS=\"-pkg api\""; \
```

Comment:
I would expose any flags as separate make vars so you could do something like this:

`make test-flake FLAKE_PKG=api FLAKE_TEST="TestFlaky" FLAKE_...`

It makes it easier to run when you don't have to deal with escaping quotes in your shell arguments.